CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## NEXT (UNRELEASED)
 
+#### Added
+
+* Treat absolute paths as absolute with respect to the `base_url`, not with respect to the file system. [PR#91]
+
```

Comment:
You need to add a reference link, markdown doesn't automatically link to github.

```suggestion
* Treat absolute paths as absolute with respect to the `base_url`, not with respect to the file system. [PR#91]

[PR#91]: https://github.com/deadlinks/cargo-deadlinks/pull/91
```